client/source/js/modules/analysis/optimization-ctrl.js

```diff
@@ -9,6 +9,7 @@ define([
 
       $scope.meta = meta;
       $scope.types = angular.copy(CONFIG.GRAPH_TYPES);
+      $scope.optimizationStatus = false;
```

Comment:
How do you feel about replacing `false` by `"not running"` & making an explicit check for "not running" in the template?

"not running" tells a bit more about whats going on and mixing booleans & strings feels a bit strange to me.